Gemfile.lock

```diff
@@ -1,53 +1,66 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (5.2.6)
```

Comment:
codeclimate is complaining about some gems - I guess we can rebase this onto the gem upgrade once we merge that and then make sure we're on the latest patch versions?

Reply:
If this is the same issue as I had on a previous patch version I think it's codeclimate being confused by the 6.0.4.x versions or something because they were CVEs that were fixed

Reply:
Yeah this passes for bundler-audit locally so I think it's fine and just a codeclimate issue